src/features/home/HomeScreenWarnings.tsx

```diff
@@ -0,0 +1,73 @@
+import { createSelector } from '@reduxjs/toolkit'
+import { useDispatch, useSelector } from 'react-redux'
+import { useLocation } from 'react-router'
+import { RootState } from 'src/app/rootReducer'
+import warningIcon from 'src/components/icons/warning.svg'
+import { Notification } from 'src/components/Notification'
+import {
+  setBackupReminderDismissed,
+  setHighValueWarningDismissed,
+} from 'src/features/settings/settingsSlice'
+import { Color } from 'src/styles/Color'
+import { fromWei } from 'src/utils/amount'
+
+const HIGH_VALUE_THRESHOLD = 25
+
+export function HomeScreenWarnings() {
+  const dispatch = useDispatch()
+  const warning = useHomeScreenWarnings()
+
+  const onDismissWarning = (warning: string) => () => {
+    const action = warning === 'backup' ? setBackupReminderDismissed : setHighValueWarningDismissed
+    if (action) dispatch(action(true))
+  }
+
+  if (!warning) return null
+
+  return (
+    <Notification
+      margin="0"
+      justify="between"
+      onDismiss={onDismissWarning(warning.key)}
+      color={Color.primaryGold}
+      textColor={Color.primaryBlack}
+      icon={warningIcon}
+      message={warning.message}
+    />
+  )
+}
+
+export const selectHomeScreenWarnings = createSelector(
+  (state: RootState) => state.settings,
+  (state: RootState) => state.wallet.balances,
+  (settings, balances) => {
+    if (!settings.backupReminderDismissed)
+      return {
+        key: 'backup',
+        message:
+          'Reminder: Be sure to copy your account key to a safe place. If you lose it, you may not be able to access your account.',
+      }
+
+    if (
+      settings.backupReminderDismissed &&
+      (fromWei(balances.cUsd) > HIGH_VALUE_THRESHOLD ||
+        fromWei(balances.celo) > HIGH_VALUE_THRESHOLD) &&
+      !settings.highValueWarningDismissed
+    )
+      return {
+        key: 'highValue',
+        message:
+          'Warning: This wallet is not recommended for high-value accounts. Please use the Valora mobile app or a Ledger wallet instead.',
+      }
+
+    return null
+  }
+)
+
+export function useHomeScreenWarnings() {
+  const warning = useSelector(selectHomeScreenWarnings)
+  const location = useLocation()
+  //ScreenFrameWithFeed for desktop doesn't differentiate between home and not home, so need to check the path
```

Comment:
It doesn't cause we didn't need to when I wrote that bit but it certainly can if it would help keep this stuff clearner. I'm fine either way

Reply:
Per above comment, I reorganized, so this is no longer necessary.